### ISSUE TITLE
IE11 bug?

### ISSUE THREAD

**Issue by jgoley** [naver/billboard.js#620]
## Description
`Unable to get property 'length' of undefined or null reference`
in `insertPointInfoDefs` method on this line (supposedly on col 38):
```
clone.id = id, clone.style.fill = "inherit", clone.style.stroke = "inherit", node.children.length && (clone.innerHTML = (0, _util.toArray)(node.children).map(function (v) {
``` 

![screen shot 2018-10-23 at 11 22 07 am](https://user-images.githubusercontent.com/4105343/47371648-2e101d00-d6b6-11e8-9b8c-d2cf89152e10.png)

This chart is in a Preact component. `bb.generate` is called in the `componentDidMount` method

## Steps to check or reproduce
Occurs when the chart is loaded in IE11.

**Comment by netil** [comment]
Hi @jgoley, the issue has been already solved - #601.
Checkout the `nightly` to get the latest solved build. :)

**Comment by jgoley** [comment]
Thanks @netil!
Apologies for the duplicate issue.
